networkx/algorithms/planarity.py

```diff
@@ -49,6 +81,11 @@ def check_planarity(G, counterexample=False):
     A counterexample is only generated if the corresponding parameter is set,
     because the complexity of the counterexample generation is higher.
 
+    See also
+    --------
+    is_planar:
+        Returns True if and only if `G` is planar.
```

Comment:
Just a minor wording suggestion that (IMO) better takes into account the context of the `check_planar` docstring. This is *very* subjective so feel free to ignore!

```suggestion
        Check for planarity without creating a `PlanarEmbedding` or counterexample.
```